docs/_posts/murat-gunay/2021-12-08-bert_token_classifier_dutch_udlassy_ner_nl.md

```diff
@@ -0,0 +1,128 @@
+---
+layout: model
+title: Dutch NER Model
+author: John Snow Labs
+name: bert_token_classifier_dutch_udlassy_ner
+date: 2021-12-08
+tags: [dutch, token_classifier, bert, ner, nl, open_source]
+task: Named Entity Recognition
+language: nl
+edition: Spark NLP 3.3.2
+spark_version: 2.4
+supported: true
+article_header:
+  type: cover
+use_language_switcher: "Python-Scala-Java"
+---
+
+## Description
+
+- This model is imported from `Hugging Face`.
```

Comment:
This model was imported from `Hugging Face`, and has been fine-tuned on Universal Dependencies Lassy dataset for Dutch language, leveraging `Bert` embeddings and `BertForTokenClassification` for NER purposes.